packages/apidom-ls/src/config/common/schema/lint/nullable--not-recommended.ts

```diff
@@ -0,0 +1,37 @@
+import { DiagnosticSeverity } from 'vscode-languageserver-types';
+
+import ApilintCodes from '../../../codes';
+import { LinterMeta } from '../../../../apidom-language-types';
+
+const nullableNotRecommendedLint: LinterMeta = {
+  code: ApilintCodes.SCHEMA_EXAMPLE_DEPRECATED,
+  source: 'apilint',
+  message: 'nullable has no special meaning, if not set on purpose use "type=null" instead',
```

Comment:
value of type must always be string even in case of the null type. This will avoid some confusion that this message might trigger
```suggestion
  message: 'nullable has no special meaning, if not set on purpose use `type="null"` instead',
```